### ISSUE TITLE
Define a new process for taking sim screenshots

### ISSUE THREAD

**Issue by amanda-phet** [phetsims/tasks#947]
There are a couple of inefficiencies in making sim screenshots, and @oliver-phet and I found an incredibly simple and seamless way to take sim screenshots using chrome. I don't want to outline every reason here for why a new process is a good idea, but the one factor holding me back from being able to take a screenshot on my high-res external monitor is I can only get to 2048x1343 (not the required 2048x1344). 

This begs the question- why do we need such a high resolution? It would be useful to know if this is being used in practice anywhere, because if it is not then it would be quite simple for anyone to take an official screenshot from their laptop. 

A quick google search tells me: For a 16" x 20" print, the image resolution should be 1600 x 1200 pixels minimum. So, for printing purposes, we certainly don't need the full retina resolution. 

There are lots of high-resolution options that we could consider that fit in the monitors the @arouinfar , @oliver-phet , and myself have for sim design: 1024x672, 1536x1008, 1792x1196.

**Comment by ariel-phet** [comment]
It will be good to hear from developers on this question as well. I feel like they were the ones that originally encouraged the highest res possible.

**Comment by oliver-phet** [comment]
Including this old issue: https://github.com/phetsims/tasks/issues/100

**Comment by amanda-phet** [comment]
Shoot! We forgot to ask this at status today!

**Comment by oliver-phet** [comment]
Untagging @kathy-phet

**Comment by amanda-phet** [comment]
Assigning myself to define the new steps

**Comment by oliver-phet** [comment]
I met with KW and LM and outlined all the steps. We're going to go with 1536x1008 for MaS since it has the same aspect ratio and no decimals. I think the procedures should live in phet-info since (hopefully) this will allow any team member to take consistent screenshots.

**Comment by oliver-phet** [comment]
@lmulhall-phet Can you write up the procedures in a new md file in phet-info?

**Comment by lmulhall-phet** [comment]
Procedure can be found here https://github.com/phetsims/QA/blob/master/doc/qa-book.md#screenshots and here https://github.com/phetsims/phet-info/blob/master/qa-team-info/Screenshots.md.

**Comment by pixelzoom** [comment]
Reopening.   The [new sim checklist](https://github.com/phetsims/phet-info/blob/master/checklists/sim_new_checklist.md) still indicates 2048x1344.

**Comment by arouinfar** [comment]
@KatieWoe @lmulhall-phet the [sim port checklist](https://github.com/phetsims/phet-info/blob/master/checklists/sim_port_checklist.md) should also be updated.

**Comment by lmulhall-phet** [comment]
Both checklists are updated with links to the new instructions... Are there any other documents that need to be updated?

**Comment by pixelzoom** [comment]
lmulhall-phet said:
> Procedure can be found here https://github.com/phetsims/QA/blob/master/doc/qa-book.md#screenshots and here https://github.com/phetsims/phet-info/blob/master/qa-team-info/Screenshots.md.

It looks to me like the latter is duplicated (copy-pasted) in the former.  Really not a good idea to duplicate this, it will definitely get out of sync at some point.  Better to put a link to the latter in the former.

**Comment by lmulhall-phet** [comment]
Will do, @pixelzoom.

**Comment by pixelzoom** [comment]
This was at one point labeled for developer meeting discussion, and apparently that happened but no notes were put in this issue.  So here's today's relevant Slack discussion on the "developer" channel:

Chris Malley [12:35 PM]
Are all devs aware that the size for screenshots was changed from 2048x1344 to 1536 x 1008 ?  This was news to me, perhaps I missed a dev discussion.  Noticed while reviewing some revised screenshots, and confusing because “new sim checklist” still says 2048x1344.

And we now have different size screenshots in master — 2048x1344 and 1536x1008.  Does having different sizes cause any downstream problems? Should they all be converted to the new size?
Previous questions probably for @matt and @jonathanolson.

Jonathan Olson [12:41 PM]
so basically the out-of-date doc can be updated, and then we are good? The aspect  ratio is the same, so it shouldn't cause any problems downstream (I recall giving the OK during a dev meeting or something)
